phpcs.xml.dist

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this required?
We are using grumpphp to carry out the checks. (Or we should be)

Reply:
@philwolstenholme had added this file in, it was in the branch. So i think we'll have to wait till he gets back

Reply:
Oh and it also runs `DrupalPractice` which Grum doesn't. Developers can choose to ignore the DrupalPractice rules if they like (it won't block commit), but it's a nice thing to be checking locally so people can learn.